src/ol/proj.js

```diff
@@ -44,9 +44,11 @@ if (ol.ENABLE_PROJ4JS) {
  * Get the resolution of the point in degrees or distance units.
  * For projections with degrees as the unit this will simply return the
  * provided resolution. For other projections the point resolution is
- * estimated by transforming the 'point' pixel to EPSG:4326,
+ * by default estimated by transforming the 'point' pixel to EPSG:4326,
  * measuring its width and height on the normal sphere,
  * and taking the average of the width and height.
+ * A custom function for a specific projection can be set on the
+ * {@link ol.proj.Projection} object for the projection.
```

Comment:
Should probably mention the name of the `getPointResolution` option and also link to `{@link ol.proj.Projection#setGetPointResolution}`.